go-controller/cmd/ovnkube-trace/ovnkube-trace.go

```diff
@@ -887,29 +887,43 @@ func getOfprotoIPFamilyArgs(protocol string, ip net.IP) (string, string, string)
 func installOvnDetraceDependencies(coreclient *corev1client.CoreV1Client, restconfig *rest.Config, podName, ovnNamespace string) error {
 	dependencies := map[string]string{
 		"ovs":       "if type -p ovn-detrace >/dev/null 2>&1; then echo 'true' ; fi",
-		"pyOpenSSL": "if rpm -qa | egrep -q python3-pyOpenSSL; then echo 'true'; fi",
+		"pyOpenSSL": "if python -c 'import ssl; print(ssl.OPENSSL_VERSION)' > /dev/null; then echo 'true'; fi",
 	}
 	for dependency, dependencyCmd := range dependencies {
-		depVerifyOut, depVerifyErr, err := execInPod(coreclient, restconfig, ovnNamespace, podName, "ovnkube-node", dependencyCmd, "")
+		verifyOut, _, err := verifyDependency(coreclient, restconfig, podName, ovnNamespace, dependency, dependencyCmd)
 		if err != nil {
-			return fmt.Errorf("dependency verification error in pod %s, container %s. Error '%v', stdOut: '%s'\n stdErr: %s",
-				podName, "ovnkube-node", err, depVerifyOut, depVerifyErr)
+			return err
 		}
-		trueFalse := strings.TrimSuffix(depVerifyOut, "\n")
-		klog.V(10).Infof("Dependency check '%s' in pod '%s', container '%s' yielded '%s'", dependencyCmd, podName, "ovnkube-node", trueFalse)
-		if trueFalse != "true" {
+		if verifyOut != "true" {
+			verifyOut, verifyErr, err := verifyDependency(coreclient, restconfig, podName, ovnNamespace, "pip3", "if type -p pip3 >/dev/null 2>&1; then echo 'true' ; fi")
+			if err != nil || verifyOut != "true" {
+				return fmt.Errorf("ovn-detrace error while installing dependency %s in pod %s, container %s. Error '%v', stdOut: '%s'\n stdErr: %s",
+					dependency, podName, "ovnkube-node", err, verifyOut, verifyErr)
```

Comment:
nit: this is already printed in `verifyDependency` so just `return err`

Reply:
done.